### ISSUE TITLE
Project Manager window briefly displays garbage

### ISSUE THREAD

**Issue by daveTheOldCoder** [godotengine/godot#62150]
### Godot version

3.5-rc4

### System information

Pop!_OS 20.04, Intel® UHD Graphics 630

### Issue description

When the Project Manager is launched, the window briefly displays old content, as if a buffer was not cleared. After a fraction of a second, the window display is normal.

This issue was not observed with 3.5-rc3.

Rebooting did not resolve the issue.

### Steps to reproduce

Launch Project Manager.

**Comment by nathanfranke** [comment]
Is this similar to https://github.com/godotengine/godot/issues/61172?

**Comment by daveTheOldCoder** [comment]
This is different from 61172 in that the "garbage" is not random noise, but old contents of a window. I would post a screenshot, but the window refreshes too quickly.

I don't know with certainty that the issue is not caused by my O/S, window manager or graphics H/W, but the issue only occurs in RC4 and not RC3. So I submitted the issue in case others are also experiencing it.

**Comment by akien-mga** [comment]
I can't reproduce, though for me starting Godot on Linux has always first displayed a window with content from what was on screen when starting the process. Then after a fraction of second the splash screen and eventually project manager appear. I don't see any difference between 3.5 RC 3 and RC 4 for this with:
```
$ inxi -CSG
System:    Host: cauldron Kernel: 5.18.5-desktop-1.mga9 x86_64 bits: 64 Desktop: KDE Plasma 5.24.4 Distro: Mageia 9 mga9 
CPU:       Info: Quad Core model: Intel Core i7-8705G bits: 64 type: MT MCP cache: L2: 8 MiB 
           Speed: 900 MHz min/max: 800/3100 MHz Core speeds (MHz): 1: 900 2: 900 3: 900 4: 900 5: 900 6: 900 7: 900 8: 900 
Graphics:  Device-1: Intel HD Graphics 630 driver: i915 v: kernel 
           Device-2: Advanced Micro Devices [AMD/ATI] Polaris 22 XL [Radeon RX Vega M GL] driver: amdgpu v: kernel 
           Device-3: Cheng Uei Precision Industry (Foxlink) HP Wide Vision FHD Camera type: USB driver: uvcvideo 
           Display: x11 server: Mageia X.org 1.21.1.3 driver: loaded: intel,v4l resolution: 1920x1080 
           OpenGL: renderer: Mesa Intel HD Graphics 630 (KBL GT2) v: 4.6 Mesa 22.1.2
```

**Comment by Zireael07** [comment]
Same here, starting Godot on Linux always displays what was on screen before starting for a fraction of a second. That's true for 3.4 and 4.x, too

**Comment by akien-mga** [comment]
The perceived difference between 3.5 RC 3 and 3.5 RC 4 might be due to async shader compilation being enabled in the editor: #62244. It makes the editor startup slower, which can mean a longer time before drawing actually first happens (and before drawing X11 will just reuse whatever was there, i.e. other windows/desktop background).

Can you reproduce the same problem in 3.5 RC 3 if you enable async shader compilation in the editor in Project Settings?

**Comment by daveTheOldCoder** [comment]
I tried toggling this setting between Synchronous and Asynchronous + Cache in both 3.5 RC 3 and 3.5 RC 4, and it had no effect on the behavior I reported above:
Project Settings >> General >> Rendering >> GLES3 >> Shaders >> Shader Compilation Mode

**Comment by akien-mga** [comment]
Can you test this build? https://downloads.tuxfamily.org/godotengine/testing/3.5-rc-android-scoped%2bndk23/Godot_v3.5-rc-android-scoped%2bndk23_x11.64.zip

It was made to test other things (Android changes) but it's from recent `3.x` branch and I suspect this bug might have been fixed since 3.5 RC 4 already. There were a few shader compilation errors that could cause something like this.

**Comment by daveTheOldCoder** [comment]
The issue does NOT occur with that build. :)

**Comment by akien-mga** [comment]
Great, then it was likely fixed by #62226 and #62251.
